lib/Web/API.hs

```diff
@@ -8,12 +8,38 @@ import Backend.Vault.Kv
 import CLI.Types
 import Coffer.Directory (Directory)
 import Coffer.Path (EntryPath, Path, QualifiedPath)
-import Data.HashMap.Strict
+import Data.Aeson.Casing (aesonPrefix, camelCase)
+import Data.Aeson.TH (deriveJSON)
+import Data.HashMap.Internal.Strict (HashMap)
 import Data.Text (Text)
 import Entry
 import GHC.Generics (Generic)
 import Servant.API
 
+----------------------------------------------------------------------------
+-- Additional types for WebAPI
```

Comment:
Please move these to a new module `Web.Types`

Reply:
Done